phoenix-core/src/main/java/org/apache/phoenix/schema/PTable.java

```diff
@@ -997,6 +997,8 @@ IndexMaintainer getIndexMaintainer(PTable dataTable, PhoenixConnection connectio
      * @throws SQLException
      */
     Set<ColumnReference> getIndexWhereColumns(PhoenixConnection connection) throws SQLException;
+
+    long getMaxLookbackAge();
```

Comment:
We're absolutely sure that every path that can possibly create a PTable from a table without a max lookback age will transparently use the default?